### PR TITLE
Feat/docker image

### LINKED ISSUE #146: Put a Docker image on Dockerhub that skips auth
People who self-host might provide other means of authentication and just want a docker container with an open port.

### PR DESCRIPTION
**Pull request by dorianim**
This fixes #146 
It is currently configured to build and push a dockerimage on every tag.

### CODE REVIEW

**Comment by dorianim** [comment]
Maybe, there should be something added in the readme, like an example docker-compose.yml.

File: .github/workflows/dockerimage.yml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN  }}
+      
+      - name: Extract metadata (tags, labels) for Docker
+        id: meta
+        uses: docker/metadata-action@98669ae865ea3cffbcbaa878cf57c20bbf1c6c38
+        with:
+          images: ghcr.io/${{ github.repository }}
```
Comment (almarklein) [inline .github/workflows/dockerimage.yml:31]: Souldn't this be called "labels"? That's how the output is referenced below.

**Review by almarklein — COMMENTED** [review]
Really nice to automate the process like this!

If I understand correctly, we still need a version where the authentication is either fully open, or password-based. Because now any attempt to auth would be rejected, right?

File: .github/workflows/dockerimage.yml (hunk 1 above)
Comment (dorianim) [reply .github/workflows/dockerimage.yml:31]: It's not only labels but also tags

Comment (almarklein) [reply .github/workflows/dockerimage.yml:31]: Oh, nevermind. I got confused by the names of the inputs for this action, and its outputs down below.

**Comment by almarklein** [comment]
Thanks!

**Pull request merged**